pkg/apply/prune/prune.go

```diff
@@ -96,6 +100,17 @@ func (po *PruneOptions) Prune(localInv *unstructured.Unstructured, localObjs []*
 	for _, clusterObj := range clusterObjs {
 		mapping, err := po.mapper.RESTMapping(clusterObj.GroupKind)
 		if err != nil {
+			u := &unstructured.Unstructured{}
+			u.SetGroupVersionKind(schema.GroupVersionKind{
```

Comment:
We shouldn't create a GVK when we don't have the version.

Reply:
Added an `ObjectMeta` field to the `PruneEvent` to solve this problem.